packages/wire-format/src/__tests__/validator.test.ts

```diff
@@ -19,21 +22,93 @@ describe('validate message', () => {
     expect(validateMessage(good.goodMessage)).toEqual(good.goodMessage);
   });
 
-  it('returns helpful error messages', () => {
-    expect(() => validateMessage(bad.dataMissing)).toThrow(
-      `Validation Error: Missing required property 'data' at root`
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const _expect: any = Object.assign(expect);
+  _expect.extend({
```

Comment:
I think this is the third time I've tried to add a custom jest matcher, and I fight with the typescript compiler every time.

I resorted to [this](https://github.com/testing-library/react-testing-library/issues/36#issuecomment-468013330).

Reply:
I did this in the `ledger-funding.test.ts` and somehow got it to work how I expected for reference